### PR TITLE
Add python versions to 3.7, 3.8 in testing workflow

### PR DESCRIPTION
**Pull request by karenc**
Also remove test job name in testing workflow

This should show the python version and OS when we look at a github job
run.

"test (ubuntu-latest, 3.6)"

### CODE REVIEW

**Review by mmulich — APPROVED** [review]

**Comment by karenc** [comment]
Let's not merge this PR until the tests are passing.

**Comment by bluemellophone** [comment]
all tests passing, ready for merge?

**Comment by karenc** [comment]
I released a new version of wbia-utool for the StopIteration fix.  Once the tests pass, this PR is ready to be merged.

**Review by mmulich — APPROVED** [review]

**Pull request merged**
